### PR TITLE
fix: mobile web login redirection

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why

Sign-in flow in iOS browser acts weird because it involves a signature request to the wallet and includes redirection to wallet app but is not user-generated action so safari ends up opening it in the browser instead of the app.

Discussed here in more detail - https://github.com/rainbow-me/rainbowkit/discussions/536

# How
- Patch package in rainbowkit to fix twitter in-app browser blank. https://github.com/rainbow-me/rainbowkit/issues/524#issuecomment-1164795047
- Show signature button after the connection is established. Discussed here in possible solutions. https://github.com/rainbow-me/rainbowkit/discussions/536

https://user-images.githubusercontent.com/23293248/175610794-d93b4203-77ba-4707-9b16-df77450e2865.mp4

# Test Plan
Test sign-in on iOS/Android browser

### CODE REVIEW

**Comment by intergalacticspacehighway** [comment]
please suggest a better text for the signature button 🥲

File: packages/app/components/login/index.web.tsx [hunk 1]
```diff
@@ -77,6 +80,16 @@ export function Login({ onLogin }: LoginProps) {
...
           </Text>
         </View>
+      ) : showSignMessage ? (
+        <View tw="py-20 px-10">
+          <Text tw="text-center text-lg dark:text-gray-400">
+            We need a signature in order to verify your identity. This won't
+            cost any gas.
+          </Text>
+          <Button tw="mt-8" onPress={verifySignature}>
+            Sign a message
```
Comment (axeldelafosse) [inline packages/app/components/login/index.web.tsx]: Maybe we could say `Sign the message` instead?

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/components/login/index.web.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/components/login/index.web.tsx]: yeah, makes sense.

**Comment by markdalgleish** [comment]
FYI I've just opened a PR to RainbowKit with a fix for the issue that was patched in this PR: https://github.com/rainbow-me/rainbowkit/pull/540

**Comment by markdalgleish** [comment]
The blank in-app browser bug is fixed in [RainbowKit v0.3.6](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.3.6).

**Pull request merged**